web-client/src/presenter/actions/setMessageAsReadAction.test.js

```diff
@@ -0,0 +1,28 @@
+import { applicationContextForClient as applicationContext } from '../../../../shared/src/business/test/createTestApplicationContext';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+import { setMessageAsReadAction } from './setMessageAsReadAction';
+
+describe('setMessageAsReadAction', () => {
+  beforeAll(() => {
+    presenter.providers.applicationContext = applicationContext;
+  });
+
+  it('should set message as read', async () => {
+    await runAction(setMessageAsReadAction, {
+      modules: { presenter },
+      props: {
+        messageToMarkRead: [
+          {
+            docketNumber: '123-45',
+            messageId: '123',
+          },
+        ],
+      },
+    });
+
+    expect(
+      applicationContext.getUseCases().setMessageAsReadInteractor,
+    ).toHaveBeenCalled();
```

Comment:
Should this test that it's been called with the right args?

Reply:
Updated!

Reply:
I see this was addressed in https://github.com/ustaxcourt/ef-cms/pull/411/commits/b4abe9f807876323c9ee41e05b324617c0796406, looks good!